pkg/docker/docker.go

```diff
@@ -346,9 +348,16 @@ func (d *stiDocker) RunContainer(opts RunContainerOptions) (err error) {
 	if opts.Stdout != nil {
 		config.AttachStdout = true
 	}
+	var hostConfig *docker.HostConfig
+	if opts.DNS != nil || opts.DNSSearch != nil {
```

Comment:
what's the docker behavior if only DNSSearch but not DNS is set here?  (ie DNS ends up getting set to nil)